src/components/Button.js

```diff
@@ -1,4 +1,6 @@
 /* eslint-disable react/prefer-stateless-function */
+/* eslint-disable import/no-extraneous-dependencies */
+
```

Comment:
- [ ] Kindly don't disable linters. If is giving errors please fix them.

Reply:
Thanks, I've just done the required changes.